.gitignore

```diff
@@ -1,2 +1,4 @@
 .flattened-pom.xml
 target
+.idea
+*.iml
```

Comment:
This should be part of your personal global gitignore file. From the [docs](https://git-scm.com/docs/gitignore):
> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig.

Reply:
Ah okay nice, ill take a look at setting this up, thanks.

Reply:
It _can_ be done that way, though it's probably a nicety to allow it in the repo too.